server/sonar-server/src/main/java/org/sonar/server/authentication/ws/LoginAction.java

```diff
@@ -68,6 +69,21 @@ public LoginAction(DbClient dbClient, CredentialsAuthenticator credentialsAuthen
   }
 
   @Override
+  public void define(WebService.NewController controller) {
+    WebService.NewAction action = controller.createAction("login")
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "login" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1192)